src/main/java/sage/utility/Display.java

```diff
@@ -0,0 +1,130 @@
+package sage.utility;
+
+import java.util.ArrayList;
+
+import sage.tasktypes.Deadline;
+import sage.tasktypes.Event;
+import sage.tasktypes.Todo;
+import sage.tasktypes.Task;
+
+public class Display {
+    private static final String LINE =
+            "█████╗█████╗█████╗█████╗█████╗█████╗█████╗\n" +
+                    "╚════╝╚════╝╚════╝╚════╝╚════╝╚════╝╚════╝";
+    private static final String LOGO = "░██████╗░█████╗░░██████╗░███████╗\n" +
+            "██╔════╝██╔══██╗██╔════╝░██╔════╝\n" +
+            "╚█████╗░███████║██║░░██╗░█████╗░░\n" +
+            "░╚═══██╗██╔══██║██║░░╚██╗██╔══╝░░\n" +
+            "██████╔╝██║░░██║╚██████╔╝███████╗\n" +
+            "╚═════╝░╚═╝░░╚═╝░╚═════╝░╚══════╝";
+
+    public String printLine() {
+        return LINE;
+    }
+
+    public void welcomeUser() {
+        String welcome = "Hello! I'm SAGE, the knowledgeable one\n"
+                + "What can I do for you today?\n";
+        System.out.println(LOGO);
+        System.out.println(welcome);
+    }
+
+    public void goodByeUser() {
+        System.out.println("Bye. Hope to see you again soon!");
+    }
+
+    /**
+     * Displays the list of tasks stored in the taskList.
+     *
+     * @param taskList The TaskList object that contains the tasks to be displayed.
+     */
+
+    /**
+     * This method prints a message to the console indicating the addition of a task to the task list and its details.
+     * The message includes the type of task, the task description and the task count after adding the task.
+     *
+     * @param userInput The input entered by the user in the form of an array of strings after character split
+     * @param taskType  The type of task being added (TODO, EVENT or DEADLINE)
+     * @param taskList  The list of tasks to which the new task is being added
+     **/
+    public void addedTask(String taskDescription, int taskCount) {
```

Comment:
```suggestion
    public void printAddedTask(String taskDescription, int taskCount) {
```
Try to make your naming more clear as to what the function does.